UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/FunctionalTests/CallEssEndPoint.cs

```diff
@@ -41,38 +42,31 @@ public async Task Setup()
         }
 
         [Test]
-        public async Task WhenICallTheExchangeSetApiWithValidProductIdentifiers_ThenValidrequestedProductCountAndexchangeSetCellCountIsReturned()
+        public async Task WhenICallTheExchangeSetApiWithValidProductIdentifiers_ThenANonZerorequestedProductCountAndexchangeSetCellCountIsReturned()
         {
-            var productIdenResponse = await getproductIdentifier.GetProductIdentifiersDataAsync(ESSAuth.EssApiUrl, productIdentifiers, EssJwtToken);
-            Assert.That((int)productIdenResponse.StatusCode, Is.EqualTo(200), $"Incorrect status code is returned {unpResponse.StatusCode}, instead of the expected status 401.");
-
-            dynamic prodId_message = await CommonHelper.DeserializeAsyncResponse(productIdenResponse);
-            Assert.That(prodId_message._links.exchangeSetBatchStatusUri.href, Is.Not.Null,"Exchange Set Batch Status Uri is null");
-            Assert.That(prodId_message._links.exchangeSetBatchDetailsUri.href, Is.Not.Null,"Exchange Set Batch Deatils Uri is null");
-            Assert.That(prodId_message._links.exchangeSetFileUri.href, Is.Not.Null,"Exchange Set File Uri is null");
-            Assert.That((int)prodId_message.requestedProductCount, Is.EqualTo(3155), $"Incorrect Requested Product Count is returned {prodId_message.requestedProductCount}, instead of the expected status 3155.");
-            Assert.That((int)prodId_message.exchangeSetCellCount, Is.EqualTo(3155), $"Incorrect Exchange Set Cell Count is returned {prodId_message.exchangeSetCellCount}, instead of the expected status 3155.");
+            var apiResponse = await getproductIdentifier.GetProductIdentifiersDataAsync(ESSAuth.EssApiUrl, productIdentifiers, EssJwtToken);
+            Assert.That((int)apiResponse.StatusCode, Is.EqualTo(200), $"Incorrect status code is returned {unpResponse.StatusCode}, instead of the expected status 200.");
+
+            //verify model structure
+            await apiResponse.CheckModelStructureForSuccessResponse();
         }
 
         [Test]
         public async Task WhenICallTheExchangeSetApiWithInValidProductIdentifiers_ThenValidrequestedProductCountAndlessexchangeSetCellCountIsReturned()
```

Comment:
Please capitalise the first letter of each word (Requested, Less, Exchange)

Reply:
Hi Jason,

The fix has been applied. Please have a look.

Thanks,
Mayuresh Gawde